pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "argotlunar";
+  version = "2.1.0";
+
+  src = pkgs.fetchgit {
+    url = "https://github.com/mourednik/argotlunar";
+    rev = "1527d5dd753a7af32600ac677337469bbf1d40cd";
+    sha256 = "sha256-U20ZUFrck/Y3GwqEnPZewnUN1YIFucgNsJJTMtWY3wE=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...`. Both currently work, but the former will make it easier if the algorithm ever needs to change.